### ISSUE TITLE
More than 2 Extruders

### ISSUE THREAD

**Issue by GSFSMeadMan** [makerbase-mks/MKS-TFT#276]
How do we configure for more than 2 extruders?  I have a 3 extruder setup, but I'm building a quad.

**Comment by ksbhatt89** [comment]
Hi GSFSMeadMan,

Could you get the answer?

**Comment by GSFSMeadMan** [comment]
Crickets....

**Comment by makerbase-mks** [comment]
Now the touch screen only supports dual nozzles.

**Comment by gysnst2** [comment]
Yes but how about virtual extruders and mixing.  Plus i can only get one extruder to install up on the Tft. Even after updating to 3.0.3 and setting 2 extruders in the firmware.

**Comment by makerbase-mks** [comment]
For now, the touch screen is just a normal double extrusion head display. Custom firmware may be required if three heads are required

**Comment by GSFSMeadMan** [comment]
is the source available for us to create a custom firmware?
